src/modules/mavlink/mavlink_messages.cpp

```diff
@@ -4204,11 +4202,12 @@ class MavlinkStreamDistanceSensor : public MavlinkStream
 				break;
 			}
 
-			msg.orientation = dist_sensor.orientation;
-			msg.min_distance = dist_sensor.min_distance * 100.0f; /* m to cm */
-			msg.max_distance = dist_sensor.max_distance * 100.0f; /* m to cm */
-			msg.current_distance = dist_sensor.current_distance * 100.0f; /* m to cm */
-			msg.covariance = dist_sensor.variance * 1e4f; // m^2 to cm^2
+			msg.current_distance = dist_sensor.current_distance * 1e3f; // m to cm
+			msg.id               = dist_sensor.id;
+			msg.max_distance     = dist_sensor.max_distance * 1e3f;     // m to cm
```

Comment:
This is wrong. Should be 1e2f

Reply:
Same for all instances of m to cm

Reply:
Fixed and rebased with current master. Thanks for catching that stupid mistake.